website/docs/ProtractorMigration.md

```diff
@@ -0,0 +1,108 @@
+---
+id: protractor
+title: From Protractor
+---
+
+This tutorial is for people who are using Protractor and want to migrate their framework to WebdriverIO. It was initiated after the Angular team [has announced](https://github.com/angular/protractor/issues/5502) that Protractor won't be longer supported. WebdriverIO has been influenced by a lot of Protractors design decisions which is why it is probably the closest framework to migrate over. The WebdriverIO team appreciates the work of every single Protractor contributor and hopes that this tutorial makes the transition to WebdriverIO easy and straightforward.
+
+While we would love to have a fully automated process for this the reality looks different. Everyone has a different setup and uses Protractor in different ways. Every step should be seen as guidance and less like a step by step instruction. If you have issues with the migration, don't hesitate to [contact us](https://github.com/webdriverio/codemod/discussions/new).
+
+## Setup
+
+The Protractor and WebdriverIO API is actually very similar, to a point where the majority of commands can be rewritten in an automted way through a [codemod](https://github.com/webdriverio/codemod). For this tutorial we use a Protractor [boilerplate project](https://github.com/webdriverio/protractor-jasmine-typescript) and fully migrate it to WebdriverIO.
+
+To install the codemod, run:
+
+```sh
+$ npm install jscodeshift @wdio/codemod
+```
+
+#### Commits:
+
+- _install codemod_ [[0fdc214]](https://github.com/webdriverio/protractor-jasmine-typescript/commit/0fdc2146bd74e5ff9e8478c2836ae9454b0e2fb1)
+
+## Strategy
+
+There are many migration strategies. Depending on the size of your team, amount of test files and the urgency to migrate you can try to transform all tests at once or file by file. Given that Protractor will continued to be maintained until Angular version 15 (end of 2022) you still have enough time. You can have Protractor and WebdriverIO tests running at the same time and start writing new tests in WebdriverIO. Given your time budget you can then start migrating the important test cases first and work your way down to tests you might even can delete.
+
```

Comment:
Is there also something that WDIO could advice regarding how to migrate the project, like:
- create a new repo and copy stuff over
- create a feature branch
- and so on

Reason for asking is that I've had a lot of questions from Sauce customers who needed to migrate from V4 V5 and didn't know what the best approach would be. Not everybody has the a good overview what would be the best.
Second reason for mentioning this is that later the doc refers to a `package.json` file and if they start a clean project they should init it with `npm init`

Reply:
This topic could fill a whole article itself IMO and it is impossible to say "this is how you should do it" because every situation will be different. Maybe there is a nice article like: https://www.smashingmagazine.com/2019/09/frankenstein-migration-framework-agnostic-approach-part-1/ which can provide additional information that we can link to.